crates/store/re_types/definitions/rerun/archetypes/points3d.fbs

```diff
@@ -11,7 +11,7 @@ namespace rerun.archetypes;
 table Points3D (
```

Comment:
As discussed, this is probably no the right archetype to initially support. Either Point2D or some custom type.